src/saltlogfiles

```diff
@@ -15,4 +15,4 @@ done
 
 section_header "Supportconfig Plugin for Salt, v${SVER}"
 
-plog_files 1000 /var/log/salt/minion /var/log/salt/master /var/log/salt/key
+plog_files 1000 /var/log/salt/minion /var/log/salt/master /var/log/salt/key /var/log/salt/broker
```

Comment:
and api ? If we add all we should add that as well.

Reply:
That's true! I also noticed `/var/log/salt/ssh` file so I'm adding it here as well.

Reply:
+1